### ISSUE TITLE
Default query with no startDate is empty

### ISSUE THREAD

**Issue by eydrian** [metaodi/tecdottir#32]
Since a while it seems that the default queries return empty results. Only by specifying a startDate that is in the past (multiple days) the request returns data.
My integration is broken for a while but did not find the time to check. 
Is there a change needed in order to get the current values?

empty results:

```
curl -X 'GET' \
  'https://tecdottir.herokuapp.com/measurements/tiefenbrunnen?sort=timestamp_cet%20desc&limit=500&offset=0' \
  -H 'accept: application/json'
```

compared to:

```
curl -X 'GET' \
  'https://tecdottir.herokuapp.com/measurements/tiefenbrunnen?startDate=2022-08-05&sort=timestamp_cet%20desc&limit=500&offset=0' \
  -H 'accept: application/json'
```

**Comment by metaodi** [comment]
Unfortunately the data provider stopped returning data, so the latest available data is from a couple days ago.
I'm working on it, and hopefully this issue is resolved soon.

**Comment by metaodi** [comment]
The data is back again. But I will make sure an empty startDate will always return the latest data.

**Comment by metaodi** [comment]
Fixed in 1.4.1, now there is no default value for `startDate` or `endDate` instead the latest available data is returned.
